app/src/main/java/name/boyle/chris/sgtpuzzles/GamePlay.java

```diff
@@ -864,7 +864,6 @@ private void startGameConfirmed(final GameLaunch launch, final String previousGa
 
 		currentBackend = startingBackend;
 		refreshColours();
-		gameView.resetZoomForClear();
```

Comment:
I hope deleting this is okay. Before, the function `resetZoomForClear` was called twice from `startGameConfirmed`: once here, and again via callpath `rebuildBitmap`->`rebuildBitmap`